flink-runtime/src/test/java/org/apache/flink/runtime/clusterframework/types/ResourceProfileTest.java

```diff
@@ -387,4 +392,36 @@ public void testSingletonPropertyOfAny() throws Exception {
 
 		assertSame(ResourceProfile.ANY, copiedProfile);
 	}
+
+	@Test
+	public void testToStringWithoutCpu() {
+		double notLoggedCpu = MAX_CPU_CORE_NUMBER_TO_LOG.doubleValue() + 1.0;
+		ResourceProfile resourceProfile = getResourceProfileForLogsWithCpuVal(notLoggedCpu);
+		assertThat(resourceProfile.toString(), is(getExpectedToString(resourceProfile, "")));
+	}
+
+	@Test
+	public void testToStringWithCpu() {
+		ResourceProfile resourceProfile = getResourceProfileForLogsWithCpuVal(1.0);
+		assertThat(resourceProfile.toString(), is(getExpectedToString(resourceProfile, "cpuCores=1.0, ")));
```

Comment:
I think we could simplify the assertion a bit by testing whether the string contains `cpuCores` and a configured memory.

Reply:
Alright, I would just make sure that the right positions of commas are covered